fe/fe-core/src/main/java/com/starrocks/load/loadv2/BrokerLoadJob.java

```diff
@@ -414,8 +415,16 @@ public void updateProgess(TReportExecStatusParams params) {
         try {
             super.updateProgess(params);
             if (!loadingStatus.getLoadStatistic().getLoadFinish()) {
-                progress = (int) ((double) loadingStatus.getLoadStatistic().totalSourceLoadBytes() /
-                        loadingStatus.getLoadStatistic().totalFileSize() * 100);
+                if (jobType == EtlJobType.BROKER) {
+                    progress = (int) ((double) loadingStatus.getLoadStatistic().sourceScanBytes() /
+                            loadingStatus.getLoadStatistic().totalFileSize() * 100);
+                    LOG.info("progress: {}, sourceScanBytes: {}, totalFileSize: {}",
```

Comment:
If we already show it in the terminal, it's nonsense to print in the log

Reply:
If we use the sourceScanBytes / totalFileSize, the number may be larger than 1?

Reply:
maybe, progress larger than 100 will reset to 99.